packages/dappmanager/test/integration/hostScripts.test.int.ts

```diff
@@ -28,8 +28,7 @@ describe.only("Host scripts", () => {
 
   it("Should execute security updates", async () => {
     const securityUpdate = await shell(
-      `sudo bash ${hostScriptsPath}/security_update.sh`,
-      { noTimeout: true }
+      `sudo bash ${hostScriptsPath}/security_update.sh`
```

Comment:
Why is sudo necessary here? Could you add a comment explaining?

Reply:
same as docker_update scripts. Will wrtie logs to a root path, where all dappnode stuff is:
```/usr/src/dappnode/logs```

I do not see it critical since will be run on github actions

We could solve this by creating an issue to only run these tests (critical tests that are susceptible to touch host machine) to be executed only on github actions